pypeit/slittrace.py

```diff
@@ -721,140 +723,296 @@ def slit_spat_pos(left, right, nspat):
         nspec = left.shape[0]
         return (left[nspec//2,:] + right[nspec//2,:])/2/nspat
 
-    def assign_maskinfo(self, sobjs, plate_scale, TOLER=1.):
+    def mask_add_missing_obj(self, sobjs, expected_objpos, fwhm, median_off, slits_left, slits_right):
```

Comment:
This code should replace our current manual object infrastructure, and should not require deimos specific mask design information to run. That can easily be done by checking whether that information exists, but the more daunting task is getting rid of the old manual extraction infrastructure.

Reply:
I disagree.  These have very different inputs and control flows.
And, as noted, it is a bit too daunting.